### ISSUE TITLE
Will not compile .pug to HTML unless "Generate-minified-HTML" setting is enabled

### ISSUE THREAD

**Issue by jasper-zanjani** [Wscats/compile-hero#64]
Compile Hero will not compile .pug files to HTML unless the **`generate-minified-html`** option is enabled. Unchecking this feature should still allow Compile Hero to compile to normal HTML.

## Steps to reproduce
```json
  "compile-hero.disable-compile-files-on-did-save-code": false,
  "compile-hero.generate-minified-html": true,
```
Saving **test.pug** or manually clicking "Compile files" from the context menu on the File Explorer compiles test.pug to two HTML files in the specified output directory:
```
dist/
├── test.html
└── test.min.html

0 directories, 2 files
```
Changing **`generate-minified-html`** to false prevents Compile Hero from compiling the pug file at all. The specified output directory is not populated or generated. Removing the setting from the settings.json entirely also results in no HTML output.

This behavior is consistent across Windows, OS X, and Linux environments.

**Comment by Wscats** [comment]
> Compile Hero will not compile .pug files to HTML unless the **`generate-minified-html`** option is enabled. Unchecking this feature should still allow Compile Hero to compile to normal HTML.
> 
> ## Steps to reproduce
> ```json
>   "compile-hero.disable-compile-files-on-did-save-code": false,
>   "compile-hero.generate-minified-html": true,
> ```
> 
> Saving **test.pug** or manually clicking "Compile files" from the context menu on the File Explorer compiles test.pug to two HTML files in the specified output directory:
> 
> ```
> dist/
> ├── test.html
> └── test.min.html
> 
> 0 directories, 2 files
> ```
> 
> Changing **`generate-minified-html`** to false prevents Compile Hero from compiling the pug file at all. The specified output directory is not populated or generated. Removing the setting from the settings.json entirely also results in no HTML output.
> 
> This behavior is consistent across Windows, OS X, and Linux environments.

I changed the settings according to your description, but the html is still normal output, can you record a video and let me see. like this: https://imgur.com/bddGTW0

**Comment by Bestulo** [comment]
I installed Compile Hero, tried to save a .pug file and the same thing happened. It wouldn't save, even though it said "Compilation Successed!" (succeeded, btw). I googled, came upon this thread, enabled minified html and it compiled successfully.

**Comment by jasper-zanjani** [comment]
that's true, I wasn't sure if "successed" was some sort of joke but if it's not meant to be cute it should be fixed

**Comment by Wscats** [comment]
> successed

Now the successful reminder is not very friendly, because some code syntax is actually a warning rather than an error, but it will also remind failed when it warns, and subsequent versions will consider optimization.
